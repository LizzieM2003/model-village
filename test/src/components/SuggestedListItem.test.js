import React from 'react';

import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { SuggestedListItem } from './SuggestedListItem';

configure({ adapter: new Adapter() });

describe('SuggestedListItem', () => {
  const suggstedListItem = shallow(<SuggestedListItem />);

  it('renders correctly', () => {
    expect(SuggestedListItem).toMatchSnapshot();
  });
});
