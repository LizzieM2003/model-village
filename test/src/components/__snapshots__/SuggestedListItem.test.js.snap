// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuggestedListItem renders correctly 1`] = `[Function]`;
